spec/abstractops.html

```diff
@@ -264,6 +264,13 @@ <h1>NonNegativeModulo ( _x_, _y_ )</h1>
     </emu-alg>
   </emu-clause>
 
+  <emu-clause id="sec-temporal-constraintorange" aoid="ConstrainToRange">
+    <h1>ConstrainToRange ( _x_, _minimum_, _maximum_ )</h1>
```

Comment:
This kind of “ClampToRange” operation seems like it’d be generally useful in the spec; there's 9 instances of `min(max(x, minimum), maximum)` in there (and 1 of `max(min(x, maximum), minimum)`). any interest in making an editorial PR, in advance of Temporal?

Reply:
No interest on my side, but feel free to take the idea and run with it.